src/runtime/tests/manifest-test.ts

```diff
@@ -3659,6 +3659,87 @@ resource SomeName
       `), `Type variable ~a does not contain field 'foo'`);
     });
 
+    it('fails to parse concrete types with inline schema field of `*`', async () => {
+      await assertThrowsAsync(async () => await parseManifest(`
+          particle Foo
+            data: reads {*}
```

Comment:
it could be good to assert on the error to ensure that the thrown error is highlighting the right section of the parsed text and that the error message is legible.